pkg/store/container/container_test.go

```diff
@@ -78,7 +81,8 @@ func TestContainerStore(t *testing.T) {
 					Attempt: 1,
 				},
 			},
-			ImageRef: "TestImage-4abcd",
```

Comment:
Needs a test that does not have a StopSignal.

Needs tests or TODO + issue for tests that use different signals.

OCI spec:
```
StopSignal string, OPTIONAL

The field contains the system call signal that will be sent to the container to exit. 
The signal can be a signal name in the format SIGNAME, for instance SIGKILL or SIGRTMIN+3.
```

Reply:
Will add an integration test for this.

Reply:
Added TODO. Will send a follow up PR.